### DISCUSSION TITLE
Is there a way to send view via webhook?

### DISCUSSION THREAD

**Discussion by SanctusAnimus** [Pycord-Development/pycord#1844] [Q&A] [answered]
I'm currently constructing webhook from url, view etc from this library, but on send i get
```
discord.errors.InvalidArgument: Webhook views require an associated state with the webhook
```
Are views restricted to bots only, or can i simulate state for it? I'm not planning to do anything fancy, it's just a few URL buttons

Webhook is constructed as `Webhook.from_url(url=webhook_url, session=session, bot_token=bot_token)`
According to docs, it returns partial Webhook, and i tried fetching complete via `await webhook.fetch(prefer_auth=True)` but that changes nothing.

Any help would be appreciated!

**Comment by Lulalaby** [comment, marked as answer]
Restricted to bots

**Reply by Lulalaby** [reply]
Is Webhook.from_url part of pycord?

**Reply by SanctusAnimus** [reply]
Works fine and i only have pycord installed to work with Discord API so i guess it is
Unfortunate that it is restricted, but i'll take it
